dataverse-webapp/src/test/resources/json/testDataset.json

```diff
@@ -8,7 +8,10 @@
   "lastUpdateTime": "2019-07-10T13:51:01Z",
   "releaseTime": "2019-07-10T13:51:01Z",
   "createTime": "2019-07-10T13:24:13Z",
-  "license": "CC0",
+  "license": {
+    "@type":"CreativeWork",
+    "name":"Different licenses or terms for individual files"
+  },
```

Comment:
Can you point to the code where we check for equality?

Reply:
Tests in `ExportServiceTest` like : 
`Assert.assertEquals(UnitTestUtils.readFileToString("exportdata/schemaorg.json"), exportedDataset.get());` - since `exportedDataset` is read from `testDataset.json` ?

Reply:
Ok. We don't expect that exported dataset will be equal to native json. Do note that `exportedDataset` is going through `exportService.exportDatasetVersionAsString` which in turn will generate json-ld format.